### DISCUSSION TITLE
can't load site in regular browser

### DISCUSSION THREAD

**Discussion by FlashlightET** [civitai/civitai#282] [Q&A]
The site just stays loading forever, but works fine in an incognito window. My friend does not have this issue. This same bug happens in other browsers too (tested in chrome and brave)

**Comment by WASasquatch** [comment]
They have changed the filter system, maybe you need to clear your filters. Too bad site is so heavily JS reliant, I'd give you the clear filter link to possibly fix your acc. I miss good old PHP sites. Lol Node.JS is such a pain to troubleshoot for people.

**Comment by FlashlightET** [comment]
i cleared my cookies yesterday and tha didnt fix it
